static/js/biomedical/protein/chart.ts

```diff
@@ -205,18 +205,25 @@ const ERROR_POINT_POSITION_Y3 = 50;
 // dimension of tissue legend dots
 const LEGEND_CIRCLE_RADIUS = 7;
 
-// interaction graph offset
-const INTERACTION_GRAPH_X_OFFSET = -150;
-const INTERACTION_GRAPH_Y_OFFSET = -25;
+// protein-protein interaction graph dimensions
+const PPI_WIDTH = 600;
+const PPI_HEIGHT = 600;
 
 // style of node representations in interaction graph viz's
 const NODE_FILL_COLORS = [
-  "mistyrose",
-  "peachpuff",
   "lightCoral",
+  "peachpuff",
+  "mistyrose",
   "lightsalmon",
 ];
 
+// decrease node repulsion strength with increasing depth
+const NODE_REPULSION_STRENGTHS = [-500, -150, -30, -20];
```

Comment:
curious why the repulsion strength changes aren't even across depths & how did you come up with these numbers?

Reply:
Thanks for the review! And I guess this is just personal preference but I kind of like having some negative space around the center node and having the higher-depth nodes be clumped together on the outside.  The default repulsion strength is -20 and the -500 I got from [this stackoverflow](https://stackoverflow.com/questions/29639270/space-out-nodes-evenly-around-root-node-in-d3-force-layout) which I'll link.